src/steamship/base/request.py

```diff
@@ -6,6 +6,11 @@ class Config:
         use_enum_values = True
 
 
+class Response(CamelModel):
+    class Config:
+        use_enum_values = True
```

Comment:
Small request for a comment to explain what this line does for future readers :)